### PR TITLE
Fix navbar for desktop and mobile

### PR DESCRIPTION
**Pull request by toni-santos**
Redesign the navbar and fix smaller related issues.

### CODE REVIEW

**Review by pedronunes19 — DISMISSED** [review]
./format.sh

**Comment by pedronunes19** [comment]
Also the previous editions dropdown doesn't seem very intuitive to me compared to previous iterations but I guess it's consistent with other dropdowns

**Comment by toni-santos** [comment]
This should be ready, we should just check everything with the design team before merging.

File: resources/js/Components/HamburgerMenu.vue [hunk 1]
```diff
@@ -21,6 +21,14 @@ const props = defineProps<{
         editions: number[];
     };
 }>();
+
+watch(open, () => {
+    if (open.value) {
+        document.body.classList.add("overflow-hidden");
+    } else {
+        document.body.classList.remove("overflow-hidden");
+    }
```
Comment (Naapperas) [inline resources/js/Components/HamburgerMenu.vue]:
```suggestion
    document.body.classList.toggle("overflow-hidden");
```

Isn't this a cleaner alternative?

File: resources/js/Components/HamburgerMenu.vue [hunk 2]
```diff
@@ -92,14 +100,18 @@ const props = defineProps<{
...
+                style="grid-template-columns: repeat(auto-fill, 100px)"
+            >
                 <template
                     v-for="edition in props.options.editions"
                     :key="`navbar-edition-link-${edition}`"
                 >
                     <ResponsiveNavLink
                         :href="route('home')"
                         :active="edition === new Date().getFullYear()"
+                        class="!w-fit"
```
Comment (Naapperas) [inline resources/js/Components/HamburgerMenu.vue]: Why the `!`?

**Review by Naapperas — CHANGES_REQUESTED** [review]
Overall looks good, however the logo seems a little bit too detailed to just be a "side bar element".

Other than that, maybe we should add smooth-scrolling to the website? (most noticeable when accessing home page links)

**Review by Naapperas — DISMISSED** [review]

**Review by Naapperas — APPROVED** [review]

**Pull request merged**